kmm/kmm-post-data/src/commonTest/kotlin/com/gchristov/newsfeed/kmmpost/ReadingTimeCalculatorTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.gchristov.newsfeed.kmmpost
+
+import com.gchristov.newsfeed.kmmpostdata.util.ReadingTimeCalculator
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class ReadingTimeCalculatorTest {
+
+    @Test
+    fun returnsZeroMinutesWhenNoWords() {
+        val totalWordCount = 0;
+        runTest(totalWordCount, 0)
```

Comment:
I think parameter names here would be helpful, e.g. `runTest(totalWordCount = totalWordCount, expectedReadingTimeMinutes = 0)` to understand at a glance what `runTest` is doing.

Reply:
Applied